src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -4834,22 +4936,47 @@ export class StudyViewPageStore
                     );
                     //only invoke if there are filtered samples
                     if (chartInfo && this.hasFilteredSamples) {
-                        const result = await internalClient.fetchGenomicDataCountsUsingPOST(
-                            {
-                                genomicDataCountFilter: {
-                                    genomicDataFilters: [
-                                        {
-                                            hugoGeneSymbol:
-                                                chartInfo.hugoGeneSymbol,
-                                            profileType: chartInfo.profileType,
-                                        },
-                                    ] as any,
-                                    studyViewFilter: this.filters,
+                        let result = [];
```

Comment:
we need to avoid adding a lot of logic/bloat to the store, which already HUGE.   Lets put this code inside named helper functions, which will also document it better.   we need to explain the basic fork: type==MUTATION_EXTENDED or otherwise.

Reply:
moved to util module